<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccuracyReportDialog</class>
 <widget class="QDialog" name="AccuracyReportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>417</width>
    <height>284</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>311</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_format">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Output Format:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_outputfile">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Output File:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_outputfile">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>170</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_browse_outputfile">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>170</y>
     <width>81</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="check_confusion_matrix">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>161</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Confusion Matrix</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="check_overall_accuracy">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>161</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Overall Accuracy</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="check_kappa">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>48</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Kappa Coefficient</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="check_users_accuracy">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>48</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>User's Accuracy</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="check_producers_accuracy">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>181</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Producer's Accuracy</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="check_other">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>90</y>
     <width>181</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_other">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>90</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Other metric...</string>
   </property>
  </widget>
  <widget class="QComboBox" name="combo_format">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Excel (.xlsx)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CSV (.csv)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>TXT (*.txt)</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AccuracyReportDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AccuracyReportDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
